docs/docsite/rst/reference_appendices/faq.rst

```diff
@@ -261,6 +261,25 @@ In general the recommended workaround is to set the following environment variab
         $ export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES
 
 
+.. _macos_as_a_target_faq:
+
+Running on macOS as a target
+----------------------------
+
+When managing a system with macOS Monterey 12, macOS Ventura
+13 or above over SSH, the following error can occur:
+
+  .. error::
+        "eDSPermissionError" DS Error: -14120 (eDSPermissionError)
+
+This is a good indication that *Allow full disk access for remote users* has not been enabled.
+
+.. seealso::
+
+   For more details, check out `the official Apple user guide article
+   <https://support.apple.com/guide/mac-help/mchlp1066/mac#mchlp1b6a98a>`_.
```

Comment:
It's not a huge deal but I consider it a best practice not to link to 3rd party pages like this from the docs to avoid link rot and maintenance.

Reply:
OTOH, we'd end up having to maintain a copy of someone else's docs. I don't know which one is worse :man_shrugging: